### PR TITLE
add some hints on debuggin

### PR DESCRIPTION
**Pull request by MalloZup**
This pr create an initial skeltron for debugging we can improve this later on

### CODE REVIEW

**Comment by MalloZup** [comment]
I don't think personally we need to cover saltstack debug link . Or at least we can add it later. Imho I wanted to highlight `our` debug files. How to debug terraform/salt is imho another topic.

File: README.md [hunk 1]
```diff
@@ -44,3 +45,8 @@ These are links to find certified systems for each provider:
 - [SAP Certified IaaS Platforms for GCP](https://www.sap.com/dmc/exp/2014-09-02-hana-hardware/enEN/iaas.html#categories=Google%20Cloud%20Platform)
 
 - [SAP Certified IaaS Platforms for Azure](https://www.sap.com/dmc/exp/2014-09-02-hana-hardware/enEN/iaas.html#categories=Microsoft%20Azure) (Be carreful with Azure, **clustering** means scale-out scenario)
+
+
+## Debug:
```
Comment (arbulu89) [inline README.md]: Remove the `:` from the header, the others don't have it

File: README.md [hunk 2]
```diff
@@ -44,3 +45,8 @@ These are links to find certified systems for each provider:
...
+
+In case if you have issue, take look at the [debugging guide](doc/debug.md)
```
Comment (arbulu89) [inline README.md]: It should be:
`In the case you have some issue, take a look at the [debugging guide](doc/debug.md)`

File: doc/debug.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Debugging:
```
Comment (arbulu89) [inline doc/debug.md]: Remove the `:` from the headers

File: doc/debug.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+The goal of this guide is to provide some useful entrypoints for debug.
```
Comment (arbulu89) [inline doc/debug.md]: s/debug/debugging/` right?

File: doc/debug.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+
+Feel free to open issue with this logs, and/or analyze them accordingly.
```
Comment (arbulu89) [inline doc/debug.md]: `Feel free to open **an** issue with **these** logs, ...`

File: README.md (hunk 1 above)
Comment (MalloZup) [reply README.md]: done thx

File: README.md [hunk 6]
```diff
@@ -44,3 +45,8 @@ These are links to find certified systems for each provider:
 - [SAP Certified IaaS Platforms for GCP](https://www.sap.com/dmc/exp/2014-09-02-hana-hardware/enEN/iaas.html#categories=Google%20Cloud%20Platform)
 
 - [SAP Certified IaaS Platforms for Azure](https://www.sap.com/dmc/exp/2014-09-02-hana-hardware/enEN/iaas.html#categories=Microsoft%20Azure) (Be carreful with Azure, **clustering** means scale-out scenario)
+
+
+## Debugging
+
+In the case you have some issue, take a look at the [debugging guide]
```
Comment (arbulu89) [inline README.md]: The url has been removed

File: doc/debugging.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
+# Debugging
+
+The goal of this guide is to provide some useful entrypoints for debug.
+
+
+Feel free to open an issue with this logs, and/or analyze them accordingly.
```
Comment (arbulu89) [inline doc/debugging.md]: `s/this/these/`

File: doc/debugging.md [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+
+
+# Salt usefull logs
+
+- `/tmp/salt-deployment.log`:  before executing formula states, execute the saltstack file contained in the repo of ha-sap-terraform-deployments.
+- `/tmp/salt-formula.log`: this is the log file where the formulas salt execution is logged. (salt-formulas are not part of the github deployments project)
```
Comment (arbulu89) [inline doc/debugging.md]: I would put the `.` after the parethesis

**Comment by MalloZup** [comment]
thx adressed

**Comment by MalloZup** [comment]
thx for review

**Pull request merged**